tools/scripts/mysql_migration.py

```
@@ -552,36 +564,96 @@ def execute(self) -> tuple[int, list]:
+                logger.info(f"  ... and {len(prepared_records) - 5} more records")
+            return len(prepared_records), self.target_tables
 
         # Insert records in batches
         batch_size = 100
-        for i in range(0, len(records), batch_size):
-            batch = records[i : i + batch_size]
+        for i in range(0, len(prepared_records), batch_size):
+            batch = prepared_records[i : i + batch_size]
             values = []
-            for tenant_id, llm_factory, api_key, status, provider_id in batch:
+            for tenant_id, llm_factory, api_key, status, provider_id, api_base, instance_name in batch:
```

Ruff (0.16.1)

[warning] Loop control variable `tenant_id` not used within loop body

(B007)